be/src/runtime/hdfs/hdfs_fs_cache.cpp

```diff
@@ -31,14 +31,14 @@ static Status create_hdfs_fs_handle(const std::string& namenode, HdfsFsHandle* h
         handle->type = HdfsFsHandle::Type::S3;
         Aws::Client::ClientConfiguration config;
         config.scheme = Aws::Http::Scheme::HTTP;
-        if (!config::aws_s3_endpoint.empty()) {
-            config.endpointOverride = config::aws_s3_endpoint;
+        if (!config::object_storage_endpoint.empty()) {
```

Comment:
why we don't need to check if it is oss path?

Reply:
auctually, codes about s3 is unused in this file.

I'm discussing with @dirtysalt now about how to add FS cache for object storage.

Reply:
remove these codes directly after discussion